tests/conftest.py

```diff
@@ -55,6 +76,30 @@ def dbt_profile_data(dbt_profile_target, profiles_config_update):
     return profile
 
 
+class SafeSecretDumper(SafeDumper):
+    """
+    A custom dumper that understands the Secret class
+    """
+
+    def represent_data(self, data):
+        if isinstance(data, Secret):
+            return self.represent_scalar('tag:yaml.org,2002:str', data.value)
```

Comment:
Please add a comment to explain this formatting. What would be the resulting example for a secret here?

Reply:
Found a better function to use and also added a comment.

Reply:
Here we dump the actual secret into the `profiles.yml` config file for DBT. Everywhere else it would be obscured.